faq-bundle/src/Resources/contao/dca/tl_faq.php

```diff
@@ -486,6 +523,25 @@ public function checkPermission()
 		}
 	}
 
+	/**
+	 * Remove the meta fields if the FAQ category does not have a jumpTo page
+	 *
+	 * @param DataContainer $dc
+	 */
+	public function removeMetaFields(DataContainer $dc)
+	{
+		$objFaqCategory = $this->Database
+			->prepare('SELECT c.jumpTo FROM tl_faq_category c LEFT JOIN tl_faq f ON f.pid=c.id WHERE f.id=?')
+			->execute($dc->id);
+
+		if (!$objFaqCategory->jumpTo)
+		{
+			PaletteManipulator::create()
+				->removeField(array('pageTitle', 'robots', 'description', 'serpPreview'), 'meta_legend')
```

Comment:
@dmolineus I have adjusted this to remove all meta fields instead of just the `serpPreview` field when the FAQ category does not have a jumpTo page. Does this make sense to you as well?

Reply:
LGTM